### PR TITLE
[BH-1443] Lower CPU freq before going to SNVS

### PR DESCRIPTION
**Pull request by paweljonskim**
Going to SNVS requires usage of periph clock
That reuires further cpu configuration changes
Easiest way is to just change cpu freq to level 2

### CODE REVIEW

File: module-bsp/board/rt1051/bellpx/board.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace
 {
     using namespace drivers;
 
-    void board_power_off()
+    void board_exit_power_off()
```
Comment (swichu91) [inline module-bsp/board/rt1051/bellpx/board.cpp:13]: How about simple `power_off`?

File: module-bsp/board/rt1051/bellpx/board.cpp [hunk 2]
```diff
@@ -27,7 +27,7 @@ namespace
         SNVS->LPCR |= SNVS_LPCR_TOP(1); /// Enter SNVS mode
     }
 
-    void board_reset()
+    void board_exit_reset()
```
Comment (swichu91) [inline module-bsp/board/rt1051/bellpx/board.cpp:30]: Is it exit or reset? ;) How about simple `reset`?

**Review by mjan-dev — APPROVED** [review]

File: module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp [hunk 3]
```diff
@@ -28,6 +28,7 @@ namespace bsp
 
     int32_t RT1051LPMCommon::PowerOff()
     {
+        SetCpuFrequency(bsp::CpuFrequencyMHz::Level_2);
```
Comment (swichu91) [inline module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp:31]: Shouldn't it be placed in the Bell specific board exit?

Comment (paweljonskim) [reply module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp:31]: well if we want this fix only for Harmony then okay :)

Comment (swichu91) [reply module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp]: From what I know, Pure does not use SNVS.

Comment (mjan-dev) [reply module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp]: Yes, Pure indeed doesn't use SNVS.

**Review by swichu91 — APPROVED** [review]

**Pull request merged**